README.md

```diff
@@ -2,94 +2,93 @@
 
 [![Version](https://vsmarketplacebadge.apphb.com/version/ms-azuretools.vscode-azureappservice.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice) [![Installs](https://vsmarketplacebadge.apphb.com/installs-short/ms-azuretools.vscode-azureappservice.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice) [![Build Status](https://dev.azure.com/ms-azuretools/AzCode/_apis/build/status/vscode-azureappservice)](https://dev.azure.com/ms-azuretools/AzCode/_build/latest?definitionId=5)
 
-App Service is Azure's fully-managed Platform as a Service (PaaS) that let's you deploy and scale web, mobile, and API apps. Use the Azure App Service extension for VS Code to quickly create, manage, and deploy your websites.
+App Service is Azure's fully-managed Platform as a Service (PaaS) that let's you
+deploy and scale web, mobile, and API apps. Use the Azure App Service extension
+for VS Code to quickly create, manage, and deploy your websites.
 
-Check out this [deployment tutorial](https://code.visualstudio.com/tutorials/app-service-extension/getting-started) to get started with deploying a Node.js Express app to Azure App Service on Linux.
+Visit the [wiki](https://github.com/Microsoft/vscode-azureappservice/wiki) for
+information on advanced features such as Remote Debugging.
 
-## Features
+## Installation
 
-* Create new web apps/deployment slots
-* Deploy to your web apps/deployment slots
-* Stream your logs to VS Code's output window
-* Add and edit environment variables (app settings)
-* Browse sites across all of your Azure subscriptions
-* Browse to the Azure Portal for advanced tasks, such as scaling
+1. Download and install the Azure App Service extension for Visual Studio Code
+    > If you're interested in deploying single page web apps or progressive web apps (something without an express server), install the [Azure Storage extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestorage)
```

Comment:
This is early for precisely this reason. If users are going to fail (e.g. deploying an Angular app), I want to put them on the right path as quickly as possible.

Reply:
If you're trying to put them on the right path, I feel like "install the Azure Storage extension" isn't that helpful. I feel like it could mention the storage extension, but it should link to a tutorial instead of just the readme. The current wording almost makes it seem like I'm supposed to install storage _in addition to_ app service

Reply:
Fair enough though I plan to update the getting started docs for storage next. Linking to the extension will land them right into the guide to deploying a static app via storage. Would t help to add an "instead" at the end?

Reply:
I guess I feel like we're taking this too far in terms of making the README a tutorial. At the end of the day, a tutorial [like this](https://code.visualstudio.com/tutorials/app-service-extension/getting-started) is still the gold standard and where we should drive the vast majority of our users IMO. It's specifically designed as a tutorial (with helpful things like the "I ran into an issue" button) without all the extra confusing junk like a "Contributing section", source code, etc.

Even if you change storage to be more _like_ a tutorial, I still think linking to [the _actual_ tutorial](https://code.visualstudio.com/tutorials/static-website/getting-started) is better